src/client/containers/Home/Home.css

```diff
@@ -0,0 +1,46 @@
+.write-resume {
```

Comment:
@shruthidasarapu @KavyaReddy732 , to get ride of horizontal scrollbar we need ```width:100%``` here: ```.homepage-slogan:nth-of-type(1)```

Reply:
@shruthidasarapu also images from ``` TitleDesc``` would in perfect size if we give them ```width:83%``` & can be added to this class: ```.write-resume img``` (was made by @DiegoRuiz454 )

Reply:
from uploaded cv section we need some adjustments to the font-family and font size